### ISSUE TITLE
Unable to start local development using Docker Compose 2.19.0

### ISSUE THREAD

**Issue by alexintech** [grafana/oncall#2492]
### What went wrong?

**What happened**:
  - `make start` fails with an error: `service postgres is required by grafana but is disabled. Can be enabled by profiles [postgres]`

**What did you expect to happen**:
  - starting development containers

### How do we reproduce it?

1. Use docker-compose v2.19.1
2. Use `COMPOSE_PROFILES=oncall_ui,redis,grafana,engine`
3. `make init`
4. `make start`

```
COMPOSE_PROFILES=oncall_ui,redis,grafana,engine DB=sqlite BROKER_TYPE=redis SQLITE_DB_FILE=./engine/oncall.db docker compose -f docker-compose-developer.yml up --remove-orphans -d
service postgres is required by grafana but is disabled. Can be enabled by profiles [postgres]
make: *** [start] Error 1
```

### Grafana OnCall Version

v1.3.1

### Product Area

Other

### Grafana OnCall Platform?

Docker

### Anything else to add?

I suppose it is caused by [this fix](https://github.com/docker/compose/pull/10602) and now `depends_on` is required by default.

As I understand `required` field would be implemented introduced in https://github.com/compose-spec/compose-spec/pull/382 

But now using docker compose v2.18.1 is the only option.

**Comment by joeyorlando** [comment]
@alexintech thanks for pointing this out. For now I guess the best solution is to downgrade to v2.18.1.

We'll keep this issue open in case others run into the same problem.

**Comment by joeyorlando** [comment]
btw, if you're like me and pushed the "Update and restart" button on Docker for Desktop and wound up w/ `docker-compose` v2.19.0, this is how you can downgrade to v2.18.1.
1. Download the `docker-compose` binary from the v2.18.1 release [here](https://github.com/docker/compose/releases/tag/v2.18.1) (select your OS's architecture)
2. Run the following commands:
```bash
$ mv <docker-compose-binary> ~/docker-compose
$ chmod +x ~/docker-compose
$ ln -sfn ~/docker-compose ~/.docker/cli-plugins/docker-compose
$ which docker-compose 
~/docker-compose  # this should be the output
$ make start # should work
```

**Comment by joeyorlando** [comment]
closing this as fixed in #2744.

@alexintech see the PR description for more info but tldr; you now need to upgrade to `docker-compose` >= 2.20.2 (instructions on how to upgrade [here](https://github.com/grafana/oncall/pull/2744#issuecomment-1664214198)).
